test/unit/warp-routes.test.ts

```diff
@@ -17,5 +21,31 @@ describe('Warp Route Configs', () => {
       expect(warpCore).to.be.an.instanceOf(WarpCore);
       expect(warpCore.tokens.length).to.be.greaterThan(0);
     });
+
+    it(`Route ${id} has valid chain names`, () => {
+      const { chainNames } = parseWarpRouteConfigId(id);
+
+      // Verify each chain exists in registry
+      for (const chain of chainNames) {
+        expect(localRegistry.getChainMetadata(chain), `Chain ${chain} not found in registry`).to.not.be.null;
+      }
+    });
+
+    it(`Route ${id} has chain names in alphabetical order`, () => {
```

Comment:
wonder if there should also be a test of "deriving the warp ID from the config is equal to `id`", which iiuc would also have covered some of these cases